app/main/views/update.py

```diff
@@ -8,9 +8,15 @@
 from app.main.services.response_formatters import api_response
 
 
+DOC_TYPES = ['services', 'briefs']
+
+
 @main.route('/<string:index_name>/<string:doc_type>/<string:document_id>',
             methods=['PUT'])
 def index_document(index_name, doc_type, document_id):
+    if doc_type not in DOC_TYPES:
+        abort(400, "doc_type must be one of: {DOC_TYPES}")
```

Comment:
So we already have a `test_should_raise_400_on_bad_doc_type` for this view that passes on master, so we must check the doctype elsewhere?

Reply:
Interesting, the get_mapping does this check, or rather checks that a valid index exists.